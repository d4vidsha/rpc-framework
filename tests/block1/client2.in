init ::1 6000
find add2
call add2 add2
1 1
close
