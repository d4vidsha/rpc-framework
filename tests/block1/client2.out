rpc_init_client: instance 0, addr ::1, port 6000
rpc_find: instance 0, add2
rpc_find: instance 0, returned handle for function add2
rpc_call: instance 0, calling add2, with arguments 1 1...
rpc_call: instance 0, call of add2 received result 2
rpc_close_client: instance 0
